website/common/locales/en/pets.json

```diff
@@ -141,5 +141,8 @@
   "clickOnPotionToHatch": "Click on a hatching potion to use it on your <%= eggName %> and hatch a new pet!",
   "notEnoughPets": "You have not collected enough pets",
   "notEnoughMounts": "You have not collected enough mounts",
-  "notEnoughPetsMounts": "You have not collected enough pets and mounts"
+  "notEnoughPetsMounts": "You have not collected enough pets and mounts",
+  "notEnoughFood": "You have not enough food",
+  "tooMuchFood": "You try to feed too much food to your pet, action cancelled",
```

Comment:
this should be `You're trying to feed too much food to your pet, action cancelled`